### PR TITLE
LPS-126603 | master-only

### PR DESCRIPTION
**Pull request by liulukiwi**
https://issues.liferay.com/browse/LPS-126603

cc @david-gutierrez-mesa

### CODE REVIEW

**Comment by liulukiwi** [comment]
ci:forward

**Comment by david-gutierrez-mesa** [comment]
ci:forward

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 1]
```diff
@@ -66,9 +66,146 @@ definition {
...
+
+		Select(
+			locator1 = "PageEditorEditableLink#FIELD",
+			value1 = "Title");
+
+		SelectFrame(value1 = "relative=top");
+
+		PageEditor.waitForAutoSave();
+
+		ContentPagesNavigator.openEditContentPage(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:146]: Unnecessary step

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 2]
```diff
@@ -66,9 +66,146 @@ definition {
...
+			pageName = "Test Content Page Name",
+			siteName = "Test Site Name");
+
+		PageEditor.addWidget(
+			portletName = "Web Content Display",
+			targetFragmentName = "Heading");
+
+		WebContentDisplayPortlet.selectWebContent(webContentTitle = "WC WebContent Title 2");
+
+		ContentPagesNavigator.openEditContentPage(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:156]: Unnecessary step

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 3]
```diff
@@ -66,9 +66,146 @@ definition {
...
+
+		PageEditor.addWidget(
+			portletName = "Asset Publisher",
+			targetFragmentName = "Web Content Display");
+
+		AssetPublisherPortlet.configureManualAssetSelectionPG(
+			assetTitle = "WC WebContent Title 3",
+			assetType = "Basic Web Content");
+
+		ContentPagesNavigator.openEditContentPage(
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:168]: Unnecessary step

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 4]
```diff
@@ -66,9 +66,146 @@ definition {
...
+
+		User.logoutAndLoginPG(
+			userLoginEmailAddress = "userea@liferay.com",
+			userLoginFullName = "userfn userln");
+
+		Navigator.gotoSitePage(
+			pageName = "Test Content Page Name",
+			siteName = "Test Site Name");
+
+		AssertElementPresent(locator1 = "Icon#EDIT_PENCIL");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:182]: Unnecessary step

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 5]
```diff
@@ -66,9 +66,146 @@ definition {
...
+
+				AssertElementNotPresent(
+					key_menuItem = "Edit",
+					locator1 = "MenuItem#ANY_MENU_ITEM");
+			}
+		}
+
+		PageEditor.clickPublish();
+
+		task ("Assert can access to web content editor via Asset Publisher in view mode") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:202]: Should be Assert cannot access to web content editor via Asset Publisher and Web Content Display in view mode

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 6]
```diff
@@ -66,9 +66,146 @@ definition {
...
+			MouseOver(
+				key_assetTitle = "WC WebContent Title 3",
+				locator1 = "AP#ASSET_ABSTRACT_TITLE");
+
+			AssertElementNotPresent(locator1 = "AP#SPECIFIC_ASSET_EDIT");
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:207]: Missing parameter

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 7]
```diff
@@ -66,9 +66,146 @@ definition {
...
+			}
+		}
+
+		PageEditor.clickPublish();
+
+		task ("Assert can access to web content editor via Asset Publisher in view mode") {
+			MouseOver(
+				key_assetTitle = "WC WebContent Title 3",
+				locator1 = "AP#ASSET_ABSTRACT_TITLE");
+
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:206]: Missing step to assert cannot access to web content editor via Web Content Display in view mode

**Comment by Tim-Cao** [comment]
@chaowang-123 
See https://github.com/liferay-echo/liferay-portal/pull/3560
View total diff https://github.com/liferay-echo/liferay-portal/pull/3560/commits/f05be22780374ea8c681504d700396f261ac3e9b

**Pull request closed without merging**